code/modules/projectiles/projectile/energy.dm

```diff
@@ -182,6 +182,26 @@
 	damage = 20
 	pass_flags = PASSTABLE | PASSRAILING
 
+	light_range = 4
+	light_color = "#ff665c"
+
+/obj/projectile/energy/blaster/disruptor/heavy
+	damage = 30
+	armor_penetration = 5
+
+/obj/projectile/energy/blaster/disruptor/explosive // Magnum variant for Nexus Trauma Team ERT + HAPT
+	damage = 50
+	armor_penetration = 10
+
+/obj/projectile/energy/blaster/disruptor/explosive/on_hit(atom/target, blocked, def_zone)
+	explosion(target, -1, 0, 2)
+	. = ..()
```

Comment:
this should not be in an ERT, because an explosion hit will instantly stun anyone and anything